packages/delisp-core/src/types.ts

```diff
@@ -37,3 +37,8 @@ export function printType(type: Type): string {
       return type.name;
   }
 }
+
```

Comment:
I don't see the benefits defining those constructor functions right now. It just adds another thing we have to define when we add new types to the language. Should we inline it for now?

Reply:
Yup. I gave it a try but it is a bit silly in this case. I'd say it's slightly easier to read but the overhead of maintaining an extra constructor is definitely not worth it here.

Maybe it will be useful for nodes that contain a lot more field in the future.